src/pocketmine/command/defaults/BanCommand.php

```diff
@@ -49,9 +49,14 @@ public function execute(CommandSender $sender, $currentAlias, array $args){
 			return false;
 		}
 
-		$name = ctype_alnum(str_replace("_", "", array_shift($args)));
+		$name = str_replace("_", "", array_shift($args));
```

Comment:
@FaigerSYS Go and read the PHP documentation instead of spamming incorrect comments.